compiler/src/main/kotlin/io/spine/protodata/renderer/InsertionPoint.kt

```diff
@@ -49,7 +64,84 @@ public interface InsertionPoint {
      * @return the line number at which the insertion point should be added.
      * @see LineNumber
      */
-    public fun locate(lines: List<String>): LineNumber
+    @Deprecated("Use locate(Text) instead.")
+    public fun locate(lines: List<String>): LineNumber {
+        val coords = locate(text(lines))
+        return when (coords.specCase) {
+            WHOLE_LINE -> LineNumber.at(coords.wholeLine)
+            INLINE -> {
+                loggerFor(InsertionPoint::class.java)
```

Comment:
Please extract it into a method. This is, obviously, an important piece of code because we have highly visible  logging in this branch. So, a method, which should be documented is highly desirable.